### PR TITLE
12.3.0 release

### PR DESCRIPTION
**Pull request by marcparadise**
12.3.0 release notes. 

I moved release notes/changelog to the top level, where most people would reasonable expect to find them.  I also moved '/omnibus/OLD_RELEASE_NOTES.md' to `/PRIOR_RELEASE_NOTES.md'

### CODE REVIEW

**Comment by marcparadise** [comment]
I do already see a couple of typos and minor corrections that I'm working on now.

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+### API Changes
+
+  * `GET $PREFIX/nodes/$NAME` will now include in the return fields
+    `policy_name` and `policy_group`.  This change is available under
+    API v0 and later
+  * `/_status` now includes information about the analytics queue state
+    when `queue_length_monitor_enabled` is set. The additional output is
+    as follows:
+
+    [..., {"analytics_queue",
```
Comment (metadave) [inline RELEASE_NOTES.md]: there are additional stats reported here: https://github.com/chef/chef-server/blob/master/src/oc_erchef/apps/oc_chef_wm/src/chef_wm_actions_queue_monitoring.erl#L221-L227

Comment (marcparadise) [inline RELEASE_NOTES.md]: Thanks, will update.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,9 +1,175 @@
...
+  * Only try to index `policy_name` and `policy_group` if their values are not undefined.
+  * Added throw to `chef_index_expander:expand/3` when key passed with undefined value.
+  * Added `efast_xs` to relx section of rebar.config.
+* `dvm` [516](https://github.com/chef/chef-server/pull/516) - forward-ssh-agent
+  * Occasionally we want to clone private repositories inside the `dev-vm`.
+    The forwarded `ssh-agent` makes this easier since the user can add their
+    github ssh key to their agent and it will be available inside the VM.
+* `dvm`, `omnibus`, `erchef` [520](https://github.com/chef/chef-server/pull/520) - direct_solr_writes
+  * add support for immediate data commits to `chef_solr`, bypassing the rabbit queue and expander process.
+    Enable this by setting `chef_solr4['search_queue_mode']` to `batch`.
```
Comment (stevendanna) [inline CHANGELOG.md]: chef_solr4 -> opscode_erchef

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,9 +1,175 @@
...
+* `omnibus` [524](https://github.com/chef/chef-server/pull/524) - cleanup-static-nginx-files
+  * Make the default index.html message more informative.
+  * delete unused javsacript files from nginx deploy
+* `omnibus` [536](https://github.com/chef/chef-server/pull/536) - master
+  * Don't consider `opscode-chef-mover` or any other hidden service status when checking `ha-status`. This is based on assumption the `opscode-chef-mover` service is only used during an upgrade, and does not need to be running all of the time.
+* `erchef` [528](https://github.com/chef/chef-server/pull/528) - spurious-status-400s
+  * Fix bug where persistent clients would receive HTTP `400`
+    after successful calls to the /_status endpoint.
+* `erchef` [529](https://github.com/chef/chef-server/pull/529) - remove-chef-otto
+  * Remove unused `chef_otto`.hrl
```
Comment (stevendanna) [inline CHANGELOG.md]: `chef_otto`.hrl -> `chef_otto.hrl`

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,9 +1,175 @@
...
+  * Mark `policy/policy` group validation specs with `:validation`.
+* `oc-chef-pedant` [614](https://github.com/chef/chef-server/pull/614) - pedant\_add\_seed\_option
+  * Rspec by default runs tests in a random order, which normally is
+    good. However sometimes bugs manifest themselves as state leftover from
+    prior tests, and it's hard to sort those out when the order changes
+    every time.
+    Add a --seed flag to pedant to set the rspec seed value.
+* `chef-mover` [613](https://github.com/chef/chef-server/pull/613) - no-eunit-on-vendored-code
+  * do not run intermittently failing tests on dependencies
+    that we can't change for `backward-compatibility` reasons.
```
Comment (stevendanna) [inline CHANGELOG.md]: No `` around backward-compatibility here.

File: RELEASE_NOTES.md [hunk 5]
```diff
@@ -0,0 +1,225 @@
...
+
+### Chef Server
+  * `policy_name` and `policy_group` are now node attributes that will be
+    returned with the node object, and can be used in node searches.
+  * Search Improvements:
+    *  Elasticsearch is now supported as a search provider.  See 'Search'
+    * You can now provide your own Solr instance instead of using the one
+      bundled with chef-server.
+    * It is now possible for Chef Server to post updates directly to the
+      search provider for real-time search index updates, bypassing rabbitmq
```
Comment (stevendanna) [inline RELEASE_NOTES.md]: Not sure we should say real-time here.  There is still solr/elasticsearch indexing delay.

File: RELEASE_NOTES.md [hunk 6]
```diff
@@ -0,0 +1,225 @@
...
+##### External Solr
+
+The following two configuration items have been added to support
+external instances of solr or elasticsearch:
+
+    opscode_solr4['external'] = false
+    opscode_solr4['external_url'] = nil
+
+When `external` is false, a local solr instance is configured and
+used for search indexing.  When it is true, we assuming a solr
```
Comment (stevendanna) [inline RELEASE_NOTES.md]: we assuming -> we assume

File: RELEASE_NOTES.md [hunk 7]
```diff
@@ -0,0 +1,225 @@
...
+Enable this as follows:
+
+    opscode_erchef['search_provider'] = "elasticsearch"
+    opscode_solr4['external'] = true
+    opscode_solr4['external_url'] = "https://PATH_TO_ELASTICSEARCH"
+    opscode_erchef['search_queue_mode'] = "batch"
+
+NOTE: Chef Server will not provision an elasticsearch server.
+To use elasticsearch, add the following settings to `chef-server.rb`
+after the server has been provisioned:
```
Comment (stevendanna) [inline RELEASE_NOTES.md]: The necessary config is actually above this paragraph.

File: CHANGELOG.md (hunk 4 above)
Comment (marcparadise) [inline CHANGELOG.md]: That's what I get for regexing things to escape ;)

File: RELEASE_NOTES.md (hunk 5 above)
Comment (marcparadise) [inline RELEASE_NOTES.md]: 'soft real time'...

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,9 +1,175 @@
...
+  * Changes to allow username for password changes
+* `omnibus` [555](https://github.com/chef/chef-server/pull/555) - gather-log-updates
+  * `gather-logs` updates
+* `oc-id` [563](https://github.com/chef/chef-server/pull/563) - oc-id-hosted-copy
+  * Just call it "Chef account" and "Chef username" and put it into the `i18n`
+    config.
+* `omnibus` [579](https://github.com/chef/chef-server/pull/579) - chef-server-ctl-proxy
+  * This configuration file is used by `chef-server-ctl` to talk to the API
+    locally. Proxy configs in the users environment often cause problems
+    because the LB VIP is almost always `127`.0.0.1, which causes the proxy to
```
Comment (joshbrand) [inline CHANGELOG.md]: IP is mis-escaped

File: CHANGELOG.md [hunk 9]
```diff
@@ -1,9 +1,175 @@
 # Chef Server Changelog
 
-# 12.2.1 (pending)
+# 12.3.0 (2015-11-12)
 
 ## Updated components
   * postgresql (9.2.10 -> 9.2.14)
+  * rubygems (1.8.24 -> 2.4.5)
+  * bundler (1.5.3 -> 1.10.6)
+  * postgresql92 (9.2.10 -> 9.2.14)
```
Comment (irvingpop) [inline CHANGELOG.md]: minor nitpick: Looks like Postgres is listed twice - once as `postgresql` and once as `postgresl92'

Comment (stevendanna) [inline CHANGELOG.md]: Also, I seem to remember redis in the manifest diff?

Comment (stevendanna) [inline CHANGELOG.md]: Full manifest diff:

```

**Pull request merged**